### DISCUSSION TITLE
2023年规划

### DISCUSSION THREAD

**Discussion by liberhe** [Dapp-Learning-DAO/Dapp-Learning#880] [General]
1 分享：
- 聚焦defi和zk方向[分享](https://www.notion.so/dapplearning/b37a0a4ab4e646e3af5758bc977c5bc8?v=f8c3344b344d45a6b1517ba90262c308)，输出ZK密码学系列，和zkdapp方向(groth16, plonk 系列，halo2，plonky2)分享。DEFI方向继续算法稳定币以及新协议研究，DAO方向项目研究；
- 邀请更多国外项目方来DL开源大学进行分享，邀请以太坊基金会的开发人员来分享，并增加英文分享比例。
- 筛选好的视频做笔记总结，并适当传播。

2 翻译小组：
 - 寻找优秀的论文和材料，devcon视频进行翻译。

3 github仓库（目标5Kstar，目前3.2K star）：
- 完成前60 task以及整理和翻译(重点layer2和zk相关task)；
- 项目完成layer2和zk学习路线图整理，方便开发者入门和进阶zkdapp；
- 招聘全职或者兼职开发参与项目维护以及设置bounty鼓励代码贡献 

4 媒体工具以及国际化（目标10K粉，半年目标5K）：
- 组建运营团队，雇全职和兼职运营小助手
- 官网搭建（github，metamask登陆），嵌入简易demo
- 增加社区项目方联动
- 合作输出文章和总结
项目所有链接：
- https://linktr.ee/dapplearning
媒体工具：
- [discord](https://discord.com/invite/gkENMS7cKz)
- [twitter](https://twitter.com/Dapp_Learning)
- [YouTube](https://www.youtube.com/c/DappLearning)
- [公众号](https://mp.weixin.qq.com/s/Y6fYmqrS9SpEw4Nd0ocUzg)

**Comment by arc0035** [comment]
如何评估“成果输出”？是指小组每个成员达到xxx水平还是

**Reply by liberhe** [reply]
> 如何评估“成果输出”？是指小组每个成员达到xxx水平还是

分享即可
